internal/stack/serverless.go

```diff
@@ -125,6 +125,29 @@ func (sp *serverlessProvider) deleteProject(project *serverless.Project, options
 	return sp.client.DeleteProject(project)
 }
 
+func (sp *serverlessProvider) currentProjectWithClientsAndFleetEndpoint(config Config) (*serverless.Project, error) {
+	project, err := sp.currentProject(config)
+	if err != nil {
+		return nil, err
+	}
+
+	err = sp.createClients(project)
+	if err != nil {
+		return nil, err
+	}
+
+	fleetURL := config.Parameters[paramServerlessFleetURL]
+	if true {
```

Comment:
Something seems to be missing here :slightly_smiling_face: 
```suggestion
	fleetURL, found := config.Parameters[paramServerlessFleetURL]
	if !found {
```

Reply:
Oh! It was already like that previously.
Thanks! I'll update it.